bofire/data_models/strategies/predictives/botorch.py

```diff
@@ -69,13 +69,13 @@ def validate_num_raw_samples(cls, v):
             )
         return v
 
-    @model_validator(mode="after")
+    @model_validator(mode="before")
     @classmethod
     def update_surrogate_specs_for_domain(cls, values):
         """Ensures that a prediction model is specified for each output feature"""
         values["surrogate_specs"] = BotorchStrategy._generate_surrogate_specs(
             values["domain"],
-            values["surrogate_specs"],
+            values["surrogate_specs"] if "surrogate_specs" in values else None,
```

Comment:
@jduerholt, is this correct?

Reply:
Hmm, I know that it definitely works, because this is always used when one does not provide any surrogates specs. I think, that, at least in the former pydantic versions, if not provided it will be None.